x-pack/qa/rolling-upgrade/src/test/resources/rest-api-spec/test/mixed_cluster/100_enrich.yml

```diff
@@ -0,0 +1,7 @@
+---
+"Enrich stats query smoke test for mixed cluster":
+
+  - do:
+      enrich.stats: {}
+
+  - length: { coordinator_stats: 3 }
```

Comment:
@tvernum This new test [failed](https://gradle-enterprise.elastic.co/s/rjxpmmummomge/tests/qvsbxgij46xlg-gx4w5a2ll3kls) in a mixed cluster environment. Next steps are:

* Mute this test
* Merge this PR once approved
* Backport and fix the bug in 7.x and merge
* Another PR to unmute this test

But I feel it is a bit unwieldy just to stick to the backport workflow. Alternatively I could:
* Fix this bug and merge to 7.x (so 7.x will be the shim)
* Merge this PR

Please let me know your thoughts on this. Thanks!

Reply:
My personal preference is to stick with the standard process of merge to master before backporting, but I don't feel so strongly as to stop you doing what makes sense to you.

Reply:
I am happy to following the standard process. No strong opinion here. In fact, I have muted the test so it can be merged first. 
But should we get the other 7.x PR to a state where it is almost ready before merging this one? Because as soon as it is merged, a bug is introduced and it can only be fixed by merging the 7.x PR. I'd like to keep the merge time between the two PRs minimal so the bug will *not* be around for long.